packages/manager/apps/cloud/client/app/cloud/project/openstack/users/token/openstack-users-token.html

```diff
@@ -78,36 +78,36 @@ <h2 class="oui-modal__title" data-translate="cpou_token_title"></h2>
                 <span data-translate="cpou_token_title_information"></span>
             </p>
 
-            <table class="oui-datagrid">
-                <thead class="oui-datagrid__headers">
+            <table class="oui-table">
```

Comment:
Any reason to keep the component name `oui-datagrid` (found under the `Table` item in documentation ?

Reply:
Yes, `oui-datagrid` is the combination of a table, a criteria and a pagination (with all the complexity around that).
`oui-table` is simply the table. In the future, the table component will be dissociated from the datagrid in a more simple one.